src/main/resources/application.yml

```diff
@@ -87,6 +84,12 @@ spring:
         provider:
           hmpps-auth:
             token-uri: ${hmpps.auth.url}/oauth/token
+  data:
+    redis:
+      database: 5
+      timeout: 60000
+      ssl:
+        enabled: false
```

Comment:
I think this will need updating in the application-local.yml, which suggests to me that this hasn't been ran locally yet. That's probably the next step?

Reply:
TODO

Reply:
thank you